src/error/utils.rs

```diff
@@ -19,20 +18,20 @@ pub(crate) fn render_token<'a>(token: &Token<'a>) -> String {
             has_sign: signed,
             unit_value: num,
             int_value: _,
-        } => render_number(*signed, *num, &token),
+        } => render_number(*signed, *num, token),
         Token::Dimension {
             has_sign: signed,
             value: num,
             int_value: _,
             unit,
         } => format!("{}{}", render_int(*signed, *num), unit),
         Token::WhiteSpace(_) => String::from(" "),
-        Token::Comment(comment) => format!("/* {} */", comment.clone()),
+        Token::Comment(comment) => format!("/* {} */", &(*comment).clone()),
```

Comment:
Replace `&(*comment).clone()` with `comment`. I did not see comment is a `&&str` in my previous review. Sorry.